services/scabbard/libscabbard/src/migrations/diesel/sqlite/migrations/2022-06-16-131600-consensus-add-decision-ack/up.sql

```diff
@@ -13,8 +13,80 @@
 -- limitations under the License.
 -- -----------------------------------------------------------------------------
 
+PRAGMA foreign_keys=off;
+
 ALTER TABLE consensus_2pc_context_participant
   ADD COLUMN decision_ack Text NOT NULL DEFAULT 'FALSE' CHECK ( decision_ack IN ('TRUE', 'FALSE') );
 
 ALTER TABLE consensus_2pc_update_context_action_participant
   ADD COLUMN decision_ack Text NOT NULL DEFAULT 'FALSE' CHECK ( decision_ack IN ('TRUE', 'FALSE') );
+
+CREATE TABLE IF NOT EXISTS new_consensus_2pc_send_message_action (
+    action_id                 INTEGER PRIMARY KEY,
+    epoch                     BIGINT NOT NULL,
+    receiver_service_id       TEXT NOT NULL,
+    message_type              TEXT NOT NULL
+    CHECK ( message_type IN ('VOTERESPONSE', 'DECISIONREQUEST', 'VOTEREQUEST', 'COMMIT', 'ABORT', 'DECISIONACK') ),
```

Comment:
We should add underscores to these as appropriate VOTE_RESPONSE, etc.; using DECISION_ACK will cut down on a change.

Reply:
Fixed